src/Admin/MaintainAuthUsers/containers/AuthUserContainer.js

```diff
@@ -61,23 +31,8 @@ class AuthUserContainer extends Component {
           role: selectedRole.roleCode,
         },
       })
-      await this.getUserRoles(contextUser.username)
       setMessageDispatch(`Role ${selectedRole.roleName} removed`)
-    } catch (error) {
-      handleError(error)
-    }
-  }
-
-  async loadUser(username) {
-    const { contextUserDispatch, handleError } = this.props
-
-    try {
-      const user = await axios.get('/api/auth-user-get', {
-        params: {
-          username,
-        },
-      })
-      contextUserDispatch(user.data)
+      getUserRoles(contextUser.username)
```

Comment:
:sos: this line fails, since I'm just calling a function that's been exposed on the HOC without any of the context (i.e. `this`).

Reply:
with ```Uncaught (in promise) TypeError: Cannot read property 'props' of undefined
```